### DISCUSSION TITLE
No more achievements?

### DISCUSSION THREAD

**Discussion by Saphyel** [community/community#56772] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't get any new achievements, is this normal?

**Comment by mrepol742** [comment]
Maybe you haven't yet fullfill the required task
You can check this for all of the known achievements and how to get 'em
https://github.com/drknzz/GitHub-Achievements

**Reply by Saphyel** [reply]
I tried to do the YoLo but didn't work :(

**Reply by RamtinKosari** [reply]
Hi again @Saphyel 
to get yolo you shold do these steps :
1. First, you need to open the repository and go to settings.
2. Second, you need to go to the collaborators category and in this section, invite an account that you have access to the repository.
3. Third, you need to add a new branch to your repository.
4. Now, you need to add a file to the new branch you created.
5. Add the file and commit the description for it and finally commit your changes
6. Click on the green button, compare and pull request.
7. Add The invited person in step-2 as a Reviewer, and create pull request.
8. At last, Check that your Reviewer and Then click on Merge pull request.
9. Done , Now you can see that Yolo Achievement in your achievements list.

**Comment by VarunGitGood** [comment]
yeah same don't see any more badges even though requirements met

**Reply by CODEX108** [reply]
Facing similar issues

**Comment by RamtinKosari** [comment]
Hi @Saphyel 
If you're not receiving any new achievements on GitHub, there could be several reasons for this:

Lack of activity: Achievements on GitHub are usually awarded for specific actions or milestones, such as creating a repository, contributing to open-source projects, merging pull requests, and more. If you haven't been actively engaging in these activities, it's natural that you wouldn't receive new achievements.

Privacy settings: GitHub allows users to choose what information they want to share publicly, including their achievements. It's possible that you have configured your privacy settings to hide your achievements, which would prevent others from seeing them.

Technical issues: Occasionally, there may be technical issues with GitHub's system that could affect the awarding of achievements. These issues are usually temporary and get resolved by GitHub's team.

Changes in achievement criteria: GitHub periodically updates its achievement system, adding or modifying the criteria for earning achievements. It's possible that the criteria for certain achievements have changed, and you may need to meet new requirements to earn them.

To troubleshoot the issue, you can check your recent activity on GitHub and see if you've been engaging in the actions that typically earn achievements. Additionally, review your privacy settings to ensure your achievements are visible to others. If you believe there may be a technical issue, you can reach out to GitHub Support for assistance.

Also you can check this [link](https://github.com/4xmen/Get-Github-Achievements/tree/main) to know how to earn achievements

**Comment by YasinDehfuli** [comment]
If you need step by step how to get YOLO badge help, use this course:

https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.yolo.md

**Comment by Vicwxy9** [comment]
https://github.com/4xmen/Get-Github-Achievements follow this link and you will get the guided steps for it
